pandas/_typing.py

```diff
@@ -1,33 +1,33 @@
 from pathlib import Path
-from typing import IO, AnyStr, TypeVar, Union
+from typing import IO, TYPE_CHECKING, AnyStr, TypeVar, Union
 
 import numpy as np
 
-from pandas._libs import Timestamp
-from pandas._libs.tslibs.period import Period
-from pandas._libs.tslibs.timedeltas import Timedelta
+if TYPE_CHECKING:  # Use for any internal imports
+    from pandas._libs import Period, Timedelta, Timestamp
```

Comment:
I think (future PR ok), that this needs some more expl; meaning you put the import here but actually use the 'str' form of the import below.

Reply:
I can add here pretty simple